homeassistant/components/plex/media_player.py

```diff
@@ -351,7 +352,6 @@ def _clear_media_details(self):
         self._media_duration = None
         self._media_image_url = None
         self._media_title = None
-        self._media_position = None
```

Comment:
I wonder if `_media_position` should sometimes still be reset to `None`, like in `media_stop` or in `turn_off`?

Reply:
I thought about this and decided to leave it be. The component doesn't currently track or respond to playing state changes. If the server reports a new position (i.e., 0) we'll use it during the next poll.